ios/RNFastCrypto.m

```diff
@@ -111,11 +111,6 @@ + (void) handleDownloadFromClarityAndProcess:(NSString*) method
 
     NSString *addr = jsonParams[@"url"];
     NSURL *url = [NSURL URLWithString:addr];
-    if (!url) {
```

Comment:
Do we have URL validation elsewhere ?

Reply:
Nope, we didn't check it, but it is sent from js call, so it's never empty.